packages/blockchain-wallet-v4-frontend/src/services/forms/validators.js

```diff
@@ -60,9 +60,17 @@ export const validEmailNotAllowed = value =>
 export const validMnemonic = value =>
   bip39.validateMnemonic(value) ? undefined : <M.InvalidPassphraseMessage />
 
+// TODO SSO: remove with new flow
```

Comment:
should this be removed?

Reply:
Keeping this in because we use this check for 2fa reset, so we still need this error message.